rapid7_insightvm_cloud/help.md

```diff
@@ -4,11 +4,13 @@ InsightVM is a powerful vulnerability management tool which finds, prioritizes,
 
 # Key Features
 
-Search through assets in order to start scans.
+* View assets
+* Start scans
+* Get scan results
 
 # Requirements
 
-Requires an InsightVM API Key
+*Requires an InsightVM API Key
```

Comment:
```suggestion
* Requires an InsightVM API Key
* InsightVM Scan Engine(s) paired to the Insight Platform
```

Reply:
Also under resources lets add:
[Scan Engine Pairing](https://docs.rapid7.com/insightvm/scan-engine-management-on-the-insight-platform/#how-to-pair-your-scan-engines-to-the-insight-platform)